tests/test_room_resource/test_update_assigned_resource.py

```diff
@@ -0,0 +1,76 @@
+import os
+import sys
+
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.room_resource.update_assigned_resource import (
+    update_assigned_resource_query,
+    expected_update_assigned_resource_query,
+    update_with_negative_quantity,
+    update_non_existing_room,
+    update_non_existing_resource
+)
+from fixtures.room.assign_resource_fixture import (
+    assign_resource_mutation
+)
+sys.path.append(os.getcwd())
+
+
+class TestUpdateRoomResorce(BaseTestCase):
```

Comment:
Kindly change the name of this class, there is a similar name in `test_update_room_resource.py`. If both classes are testing a related feature I would recommend putting the test cases together.